deployments/test/5_rebalancing.spec.ts

```diff
@@ -233,4 +250,214 @@ describe('Deployment: Rebalancing', () => {
+      const ethDaiRebalancingSetTokenAddress = await getContractAddress('ETHDaiRebalancingSetToken');
+      ethDaiRebalancingSetToken = new web3.eth.Contract(RebalancingSetToken.abi, ethDaiRebalancingSetTokenAddress);
+    });
+
+    it('find a valid contract at the address', async () => {
+      const code = await getContractCode('BitEthRebalancingSetToken', web3);
+      expect(code.length).toBeGreaterThan(3);
+    });
+
+    it('rebalanced set should have have the correct rebalancing set token factory', async () => {
+      const rebalancingSetTokenFactoryAddress = await getContractAddress('RebalancingSetTokenFactory');
+      const receivedSetTokenFactory = await ethDaiRebalancingSetToken.methods.factory().call();
+      expect(receivedSetTokenFactory).toEqual(rebalancingSetTokenFactoryAddress);
+    });
+
+    it('rebalanced set should have the correct components', async () => {
+      const collateralSetAddress = await getContractAddress('ETHDaiInitialCollateralSet');
+      const setTokenComponents = [collateralSetAddress];
+      const receivedSetComponents = await ethDaiRebalancingSetToken.methods.getComponents().call();
+      expect(receivedSetComponents).toEqual(setTokenComponents);
+    });
+
+    it('rebalanced set should have the correct natural unit', async () => {
+      const receivedNaturalUnits = await ethDaiRebalancingSetToken.methods.naturalUnit().call();
+      expect(receivedNaturalUnits.toString()).toEqual(constants.DEFAULT_REBALANCING_NATURAL_UNIT.toString());
+    });
+
+    it('rebalanced set should have the correct proposal period', async () => {
+      const receivedProposalPeriod = await ethDaiRebalancingSetToken.methods.proposalPeriod().call();
+      expect(receivedProposalPeriod.toString()).toEqual(networkConstants.ethDaiProposalPeriod[networkName].toString());
+    });
+
+    it('rebalanced set should have the correct rebalance interval', async () => {
+      const receivedRebalanceInterval = await ethDaiRebalancingSetToken.methods.rebalanceInterval().call();
+      expect(receivedRebalanceInterval.toString()).toEqual(networkConstants.ethDaiRebalanceInterval[networkName].toString());
+    });
+
+    it('rebalanced set should have the correct whitelist contract', async () => {
+      const whiteListAddress = await getContractAddress('WhiteList');
+      const receivedWhiteListAddress = await ethDaiRebalancingSetToken.methods.componentWhiteListAddress().call();
+      expect(receivedWhiteListAddress).toEqual(whiteListAddress);
+    });
+
+    it('rebalanced set should have the correct set name', async () => {
+      const receivedSetName = await ethDaiRebalancingSetToken.methods.name().call();
+      expect(receivedSetName).toEqual('BTD ETHDai Set');
+    });
+
+    it('rebalanced set should have the correct set symbol', async () => {
+      const receivedSetSymbol = await ethDaiRebalancingSetToken.methods.symbol().call();
+      expect(receivedSetSymbol).toEqual('ETHDai');
+    });
+
+  });
```

Comment:
This looks mint

Reply:
Unsure what Aussie slang that means...but I'm assuming you want me to remove the extra spaces haha